GCMessengerSDKSample/GCMessengerSDKSample/ChatWrapperViewController.swift

```diff
@@ -199,15 +199,20 @@ extension ChatWrapperViewController: ChatControllerDelegate {
         }
     }
     
-    func showAuthenticatedSessionExpirationAlert(message: String) {
-        delegate?.didClientResponseError(message: message)
+    func showAuthenticatedSessionErrorAlert(message: String) {
+        delegate?.authenticatedSessionError(message: message)
     }
     
     func reconnectChat() {
         self.chatController.reconnectChat()
     }
     
     @objc func logout(_ sender: UIBarButtonItem?) {
+        DispatchQueue.main.async { [weak self] in
+            guard let self else { return }
+            
+            self.chatController.logoutFromAuthenticatedSession()
```

Comment:
if chatController.logoutFromAuthenticatedSession() needs to run on main thread why isn't it an internal behavior of ChatController?

Reply:
It's not needed, I revert it